src/markdown.ml

```diff
@@ -48,6 +48,45 @@ let escape_markdown_characters s =
   done;
   Buffer.contents b
 
+let print_attributes b a =
+  match a with
+  | {id = None; classes = []; attributes = []} -> ()
+  | _ ->
+    let start = ref false in
+    Printf.bprintf b "{";
+    begin
+      match a.id with
```

Comment:
Style: usually we put the `match` in the same line as the `begin` to avoid increasing indentation.